imagenet/README.md

```diff
@@ -31,10 +31,16 @@ ImageNet dataset is large and time-consuming to download. To get started quickly
 python main.py -a resnet18 --dummy
 ```
 
+To test XPU
+
+```bash
+python main.py -a resnet18 --xpu True --dummy
+```
+
 ## Multi-processing Distributed Data Parallel Training
 
 You should always use the NCCL backend for multi-processing distributed training since it currently provides the best distributed training performance.
-
+Multiprocessing is not tested yet in XPU
```

Comment:
I would reformulated the whole paragraph:
```
If running on CUDA, you should always use the NCCL backend for multi-processing distributed training since it currently provides the best distributed training performance.

For XPU multiprocessing is not supported as of PyTorch 2.6.
```